### PR TITLE
Added Project Description for ROT-13

### LINKED ISSUE #741: Add ROT13 to Algo List
I’d like to get into some number theory algorithms. This on is super easy to kick it off.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

### PR DESCRIPTION
**Pull request by auroq**
Congrats on taking the first step to contributing to the [Sample Programs in Every Language][1] repository!

For simplicity, please make sure that your pull request includes one and only one sample program and language.

## Please Complete the Following

- [ ] I would **NOT** like credit for my contribution in the article (credit will be included otherwise)
- [ ] I plan to write the article for this code snippet
- [ ] I named the pull request using `<Sample Program> in <Language>` format
- [ ] I created/updated the language README
  - [ ] I added the sample program name to the README
  - [ ] I added fun facts (i.e. debut, developer, typing, etc.)
  - [ ] I added reference link(s) to the README
  - [ ] I added solution citations when necessary (see [plagiarism][2])
- [x] I fixed #741 

## Notes

Feel free to put whatever you want here.

[1]: https://therenegadecoder.com/code/sample-programs-in-every-language/
[2]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/CONTRIBUTING.md#plagiarism

### CODE REVIEW

**Comment by jrg94** [comment]
Thanks for taking care of this! Would you mind adding yourself to the [`authors.yml`](https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/_data/authors.yml) file?

File: docs/projects/rot-13/index.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+title: ROT-13 in Every Language
+layout: default
+date: 2018-11-20
+last-modified: 2018-11-20
+featured-image:
+tags: [rot-13]
+authors:
+---
+
+ROT13 is a letter substitution cipher where every letter is replaced by the
```
Comment (jrg94) [inline docs/projects/rot-13/index.md]: There are a handful of consistency issues here with the naming of ROT13/ROT-13. I don't mind which way you go, but it should probably be consistent throughout this document.

File: docs/projects/rot-13/index.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+---
+title: ROT-13 in Every Language
+layout: default
+date: 2018-11-20
+last-modified: 2018-11-20
+featured-image:
+tags: [rot-13]
+authors:
```
Comment (jrg94) [inline docs/projects/rot-13/index.md]: Thanks for taking care of this! Would you mind adding yourself to the [`authors.yml`](https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/_data/authors.yml) file? That way this has a name to publish under.

File: docs/projects/rot-13/index.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+---
+
+ROT13 is a letter substitution cipher where every letter is replaced by the
+letter 13 letters after it alphabetically and wrapping from `Z` to `A` if necessary.
+
+    ABCDEFGHIJKLMNOPQRSTUVWXYZ -> NOPQRSTUVWXYZABCDEFGHIJKLM
+
```
Comment (jrg94) [inline docs/projects/rot-13/index.md]: You should add a paragraph about my favorite ROT13 property 😆 which is that strings return to themselves when they are encrypted again.

File: docs/projects/rot-13/index.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+## Testing
+
+The following table contains various test cases that you can use to
+verify the correctness of your solution:
+
+| Description  | Input | Output |
+|--------------|-------|--------|
+| No Input     | | "Usage: please provide a string to encrypt" |
+| Empty Input  | | "Usage: please provide a string to encrypt" |
```
Comment (jrg94) [inline docs/projects/rot-13/index.md]: Just for the sake of specifying two clearly different test cases, there should be an empty string in the input here.

File: docs/projects/rot-13/index.md (hunk 3 above)
Comment (jrg94) [reply docs/projects/rot-13/index.md]: Actually, I'm happy to do this.

File: docs/projects/rot-13/index.md (hunk 2 above)
Comment (auroq) [reply docs/projects/rot-13/index.md]: Happy to help! I have added myself.

File: docs/projects/rot-13/index.md (hunk 1 above)
Comment (auroq) [reply docs/projects/rot-13/index.md]: I decided to go with ROT-13. I think I got them all. Let me know if I missed one.

**Pull request merged**